context/microfrontends.tsx

```diff
@@ -2,34 +2,52 @@ import { ExampleTinyFrontendType } from "@tiny-frontend/example-tiny-frontend-co
 import React, { PropsWithChildren, ReactNode } from "react";
 
 export interface MicroFrontendStore {
-  ExampleTinyFrontendServer: ExampleTinyFrontendType;
+  ExampleTinyFrontendServer: ExampleTinyFrontendType | string;
 }
 
 export const initialState: MicroFrontendStore = {
-  ExampleTinyFrontendServer: () => null,
+  ExampleTinyFrontendServer: () => null
 };
 
 const MicrofrontendsContext =
   React.createContext<MicroFrontendStore>(initialState);
 
-const MicrofrontendsContextProvider: React.FC<PropsWithChildren<any>> = ({
+const getMicrofrontends = (values: MicroFrontendStore): MicroFrontendStore =>
+  Object.keys(values).reduce(
+    (acc, curr) => {
+      if (
+        // ignore this hard check
+        curr === "ExampleTinyFrontendServer" &&
+        typeof values.ExampleTinyFrontendServer === "string"
+      ) {
+        const fnWorking = "return function () { return `Hello, world` }";
```

Comment:
This one works fine if we set `ExampleTinyFrontendServer` to `new Function(fnWorking)()`